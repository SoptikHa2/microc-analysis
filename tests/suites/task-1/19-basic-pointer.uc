 main() {
   var x, y;
   x = 43;
   y = &x;
   *y = *y - 1;
   return x;
 }
