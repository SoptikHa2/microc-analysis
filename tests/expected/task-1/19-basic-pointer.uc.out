---
test: run suites/task-1/19-basic-pointer.uc
---

exit: 42
